h5p.classes.php

```diff
@@ -3314,19 +3372,18 @@ private function filter_xss($string, $allowed_tags = array('a', 'em', 'strong',
    *   An array with various meaning depending on the value of $store.
    *   If $store is TRUE then the array contains the allowed tags.
    *   If $store is FALSE then the array has one element, the HTML tag to process.
-   * @param $store
+   * @param bool $store
    *   Whether to store $m.
-   *
-   * @return
-   *   If the element isn't allowed, an empty string. Otherwise, the cleaned up
-   *   version of the HTML element.
+   * @return string If the element isn't allowed, an empty string. Otherwise, the cleaned up
+   * If the element isn't allowed, an empty string. Otherwise, the cleaned up
+   * version of the HTML element.
    */
   private function _filter_xss_split($m, $store = FALSE) {
     static $allowed_html;
 
     if ($store) {
       $allowed_html = array_flip($m);
-      return;
+      return $allowed_html;
```

Comment:
keeping return statements consistent, this value should technically never be used but this function returns a string and there for should return the string being cached